paper2remarkable/providers/_base.py

```diff
@@ -231,11 +234,17 @@ def run(self, src, filename=None):
                     return input()
 
                 if self.upload:
-                    return upload_to_remarkable(
-                        clean_filename,
-                        remarkable_dir=self.remarkable_dir,
-                        rmapi_path=self.rmapi_path,
-                    )
+                    if not self.usb_upload:
```

Comment:
Small nitpick: Style-wise a conditional may be easier to read by putting the positive clause first:
```python
if self.usb_upload:
  return upload_to_remarkable_usb(...)
else:
  return upload_to_remarkable_rmapi(...)
```